### PR TITLE
BGDIINF_SB-2454: drop Ireland support, only go for Frankfurt

### PR DESCRIPTION
**Pull request by hansmannj**
After migration to Frankfurt, obsolete files like rc_*, deploy related
scripts and config files, py2 buildspec file and git hooks became
obsolete and are deleted now. Also the README.md was updated to reflect
the Frankfurt setup.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,12 +24,10 @@ chsdi/static/doc/build/
 chsdi/static/css/extended.min.css
 chsdi/static/css/blueimp-gallery.min.css
 chsdi/static/info.json
-deploy/hooks/pre-create-code
 node_modules
 MANIFEST
 buildout_branch.cfg
 25-mf-chsdi3.conf
-deploy/deploy-branch.cfg
 deploy/conf/00-branch.conf
```
Comment (ltshb) [inline .gitignore:31]: Should this not also be removed ?

File: .gitignore [hunk 2]
```diff
@@ -26,12 +24,10 @@ chsdi/static/doc/build/
 chsdi/static/css/extended.min.css
 chsdi/static/css/blueimp-gallery.min.css
 chsdi/static/info.json
-deploy/hooks/pre-create-code
 node_modules
 MANIFEST
 buildout_branch.cfg
```
Comment (ltshb) [inline .gitignore:29]: Same for this ?

File: .gitignore (hunk 1 above)
Comment (ltshb) [reply .gitignore:31]: Below should we also not remove `rancher-*.yml` and what about `tests/varnish*` ?

**Review by ltshb — COMMENTED** [review]
About the Readme what did you do exactly ? I think it might be wise to do the following for the moment:
1. Rename README.md to README_IRELAND.md
2. Rename PYTHON3.md to README.md
We can then delete README_IRELAND.md together with the Makefile.Ireland.

File: .gitignore (hunk 2 above)
Comment (hansmannj) [reply .gitignore:29]: Good catches, thanks! I'll scan through the .gitignore again and try to find everything that I did not find in the first shot

**Comment by hansmannj** [comment]
> About the Readme what did you do exactly ? I think it might be wise to do the following for the moment:
> 
>     1. Rename README.md to README_IRELAND.md
> 
>     2. Rename PYTHON3.md to README.md
>        We can then delete README_IRELAND.md together with the Makefile.Ireland.

I kind of merged them so that the resulting README.md represents the Frankfurt setup.
But your suggestion makes sense, would make it easier to check former states, if needed. I'll do it that way

**Review by ltshb — APPROVED** [review]

**Pull request merged**